server/utils/README.md

```diff
@@ -0,0 +1,7 @@
+### **The utils directory will serve as a location for mutltiple utilty tools.  get_request_data_csv.py can be used to make an API call to fetch 311 request data from the 311 data server for a given start date and end date.** 
```

Comment:
Have you taken a look at the [preview](https://github.com/hackforla/311-data/blob/e334bd4d009d8e8cd1824e98512738cabfb8b34b/server/utils/README.md) for this file? Everything is bolded.

Reply:
Removed the bold font.